### DISCUSSION TITLE
Make AMS send email in case of fault

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6337] [Ideas]
make AMS send email incase of server fault , high cpu or high ram that we can customize . 
this is very high needed

**Comment by yashtandon113** [comment]
@brixxnatt 
Thanks for your questions.

You can get notified of high resource usage with the help of a webhook URL. Check out the below discussion.
https://github.com/orgs/ant-media/discussions/5770

I hope it helps.

**Reply by mekya** [reply]
I think we can connect some kind of emailing for a fault. Then I'm converting this to an issue.
